lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -31,6 +31,9 @@ export default function FilePickerApp({
     authToken,
     authUrl,
     courseId,
+    // It is assumed that enableLmsFilePicker is only used by LMS apps
```

Comment:
pulled this comment from the py side, I found it useful when trying to figure out how to put the client into local file picker mode.